.dockerignore

```diff
@@ -7,6 +7,8 @@ public/
 registration.yml
 tests/
 tsconfig.tsbuildinfo
+Dockerfile
+.dockerignore
```

Comment:
This is to avoid voiding the cache every time I change the docker image, but doesn't change anything to the end image